packages/amplication-data-service-generator/src/update-package-jsons.ts

```diff
@@ -24,9 +24,9 @@ export function updatePackageJSONs(
   });
 }
 
-function updatePackageJSON(module: Module, update: Record<string, any>) {
+function updatePackageJSON(module: Module, update: { [key: string]: any }[]) {
   let pkg = JSON.parse(module.code);
-  pkg = merge(pkg, update);
+  pkg = merge(pkg, ...update);
```

Comment:
In the previous cases, we used forEach (see screenshot). I just want to make sure that the behavior of merge with the spread operator will yield the same results as forEach

<img width="763" alt="image" src="https://user-images.githubusercontent.com/43705455/195254729-e7cc4b94-9655-4cdd-8083-0e88c7d70897.png">